src/contracts/adapters/manager-adapter.js

```diff
@@ -29,7 +29,7 @@ export const submitAndProcessProposal = async ({
   const configValues = numericConfigValues ? numericConfigValues : [];
   const configAclFlags = aclFlags ? new Array(...aclFlags) : new Array();
   const { extensionAddresses, extensionAclFlags } = Object.values(
-    extensions
+    extensions ?? {}
```

Comment:
out of curiousity, what does "??" ? 
I would have imagines here "||" so if extensions is undefined, it returns an empty object.

Reply:
?? returns the right operand when the left is null/undefined whereas || evaluates the operands as booleans. In this case || and ?? would have the same effect in returning the empty object, no specific reason to use it in this case. I can change it to || if that makes the code more readable